stylesheet/main.css

```diff
@@ -75,11 +90,13 @@ footer {
     box-shadow: 0 0 4px rgba(0,0,0,0.2);
 }
 
+.mugshot {
+    float: right;
+    position: relative;

```

Comment:
You need to set `position: relative` in order for the `top` declaration to have an effect.